internal/util/fetch/fetch.go

```diff
@@ -98,15 +98,15 @@ func (c Command) validate(kf *kptfilev1alpha2.KptFile) error {
 func cloneAndCopy(ctx context.Context, r *git.RepoSpec, dest string) error {
 	const op errors.Op = "fetch.cloneAndCopy"
 	p := printer.FromContextOrDie(ctx)
-	p.Printf("cloning %s@%s\n", r.OrgRepo, r.Ref)
+	p.Printf(nil, "cloning %s@%s\n", r.OrgRepo, r.Ref)
```

Comment:
having options as fist parameters and providing nil is a good sign it should not be the first paramter.

Reply:
Add a new function `OptPrintf` to accept options.